server/src/main/java/com/cloud/server/ManagementServerImpl.java

```diff
@@ -3453,6 +3491,7 @@ public long getMemoryOrCpuCapacityByHost(final Long hostId, final short capacity
         cmdList.add(IssueOutOfBandManagementPowerActionCmd.class);
         cmdList.add(ChangeOutOfBandManagementPasswordCmd.class);
         cmdList.add(GetUserKeysCmd.class);
+        cmdList.add(ChangeOfferingForVolumeCmd.class);
```

Comment:
Better to move this to the volume cmds added above.

https://github.com/apache/cloudstack/blob/3ddcf858767ff5f79986457cbbac4c856b73e0bd/server/src/main/java/com/cloud/server/ManagementServerImpl.java#L3260-L3271